backend/cfg/cfg.mli

```diff
@@ -69,6 +69,7 @@ type basic_block =
 type t = private
   { blocks : basic_block Label.Tbl.t;  (** Map from labels to blocks *)
     fun_name : string;  (** Function name, used for printing messages *)
+    fun_args : Reg.t array;  (** Function arguments *)
```

Comment:
I think a (compact) version of the following
(from the pull request description) would be
useful to keep here:
> When Cfg is constructed from Linear, this information is not available (Linear.fundecl does not have fun_args field, this information is not needed any more / has been compiled away) and so we assign an empty array as a dummy value for this field.